src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,64 @@
+package duke.command;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import duke.Ui;
```

Comment:
The [ordering](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements) of the import statements in the all the classes can be consistent so that it's easier to browse the list :-)